### ISSUE TITLE
requesting training-related code or configuration files.

### ISSUE THREAD

**Issue by aveav** [Zeying-Gong/Falcon#5]
Hello,

I am working with your project Falcon, but I have encountered some issues and would appreciate your help.

Specifically, I would like to request:
1. The complete configuration file used for training.
2. If possible, the training code, especially the parts related to setting up the training process, configuring the training environment, and defining the policy.

I have attempted to set up the training with my existing configuration, but I am still facing some difficulties. Any example code or configuration file you could provide would be incredibly helpful in assisting me with this process.

 Additional Information:
- Training Command: `python -u -m habitat_baselines.run --config-name=social_nav_v2/falcon_mp3d.yaml habitat_baselines.evaluate=False`

I would greatly appreciate it if you could provide the relevant code or configuration files.

Thank you for your assistance!

**Comment by aveav** [comment]
I would like to add a follow-up question regarding the pretrained models.

Could you please clarify how the two pretrained models, `falcon_pretrained_25.pth` and `pretrained_habitat3.pth`, were obtained? Specifically, I am interested in understanding the process behind training these models. Did you only use  Habitat-Baselines directly to train them, or was there any additional custom training procedure involved? Also, what is the key difference between these two models in terms of training data, architecture, or performance?

Thank you for your help!

**Comment by Zeying-Gong** [comment]
Thanks for your questions and for working with Falcon. Here’s a quick response to your concerns:

### Training Configuration and Code
To start training, simply set `habitat_baselines.evaluate=False` just like your command. For reproducibility:
1. As mentioned in the paper, we trained exclusively on the HM3D dataset. The MP3D results were obtained via zero-shot inference to demonstrate the model’s generalization ability.
2. The training duration was approximately 2 days (as reflected in the `wall_time` of 171933 seconds in the checkpoint’s `extra_state`). To make reproduction easier, I’ve added a one-click script for training in the repository.

### Follow-up Question on Pretrained Models
The two pretrained models differ as follows:
- **`pretrained_habitat3.pth`**: This is adapted from the HM3D open-source point navigation model trained with only depth input ([link](https://github.com/facebookresearch/habitat-matterport3d-dataset/tree/main/pointnav_comparison)). We modified it to be compatible with Habitat 3.
- **`falcon_pretrained_25.pth`**: This is fine-tuned from `pretrained_habitat3.pth` using reinforcement learning with our social navigation auxiliary tasks and future-aware reward setup.

Further details can be found in the paper, including the architecture, training data, and experimental setup.

**Comment by aveav** [comment]
"So you completely used habitat-baselines to train the model without writing custom training code, is that correct?"
